app/src/main/java/com/randomappsinc/studentpicker/home/DeleteListDialog.java

```diff
@@ -19,16 +19,21 @@
     DeleteListDialog(@NonNull Listener listener, Context context) {
         dialog = new MaterialDialog.Builder(context)
                 .title(R.string.confirm_deletion_title)
-                .content(R.string.confirm_deletion_message)
                 .positiveText(android.R.string.yes)
                 .negativeText(android.R.string.cancel)
                 .onPositive((dialog, which) ->
                         listener.onDeleteListConfirmed(position)
                 ).build();
     }
 
-    public void show(int position) {
+    // Well there is no purpose of sending the position here do you keep it for the upcoming work?
+    void presentForList(int position, String listName) {
         this.position = position;
+        String dialogContent = dialog.getContext().getString(R.string.confirm_deletion_message," \"" + listName + "\"");
```

Comment:
nit: Slight formatting improvement

```suggestion
        String dialogContent = dialog.getContext().getString(R.string.confirm_deletion_message, " \"" + listName + "\"");
```